music_assistant/common/models/enums.py

```diff
@@ -142,12 +142,12 @@ class ContentType(StrEnum):
     UNKNOWN = "?"
 
     @classmethod
-    def _missing_(cls: Self, value: object) -> Self:  # noqa: ARG003
+    def _missing_(cls, value: object) -> Self:  # noqa: ARG003
```

Comment:
```suggestion
    def _missing_(cls: type, value: object) -> Self:  # noqa: ARG003
```

I think strictly speaking it should be this ?

Reply:
well, I think it doesnt matter that much as no type checker is wining about it (guess because its still not clear haha).
Lets keep it just without the type hint